packages/fluentui/docs/src/prototypes/VirtualizedStickyTree/VirtualizedStickyTree.tsx

```diff
@@ -0,0 +1,465 @@
+import * as React from 'react';
+import { VariableSizeList as List } from 'react-window';
+import { ObjectShorthandCollection, Tree, TreeItemProps, treeTitleClassName } from '@fluentui/react-northstar';
+import StickyTreeTitle from './StickyTreeTitle';
+import { InnerElementContext, InnerElementContextType } from './context';
+
+class ItemWrapper extends React.PureComponent<{
+  index: number;
+  isScrolling?: boolean;
+  style: Object;
+  data: any;
+}> {
+  render() {
+    const { index, style, data } = this.props;
+    const item = data.renderedItems[index];
+    if (item) {
+      return data.isNonSticky(item)
+        ? React.cloneElement(item, {
+            style,
+            onTitleClick: data.onClick,
+            onFocusParent: data.handleFocusParent,
+            onFocusFirstChild: data.handleFocusFirstChild,
+            onSiblingsExpand: (event: React.SyntheticEvent<HTMLElement>, treeItemProps: TreeItemProps) => {
+              if (treeItemProps.level !== 1)
+                // don't care about onsibling Expand for sticky headers
+                data.handleOnSiblingsExpand(event, treeItemProps);
+            },
+            onFocus: (e: React.SyntheticEvent<HTMLElement>) => {
+              if (item.props.level === 1) return;
+              data.handleOnFocus(e);
+            },
+          })
+        : null;
+    }
+
+    return null;
+  }
+}
+
+const InnerElementType = ({ children, style }) => {
+  const context = React.useContext(InnerElementContext);
+  const { height, stickyItemSize, stickyTopIds, stickyBottomIds, onClickSticky } = context;
+
+  if (!children.length) return <></>;
+
+  const data = children[0]?.props?.data;
```

Comment:
You shouldn't need to validate `children[0]` since you've already done it in the above guard

```suggestion
  const data = children[0].props?.data;
```